dataeng/jobs/analytics/MonitorBigqueryEventLoading.groovy

```diff
@@ -1,36 +0,0 @@
-package analytics
```

Comment:
There's also the code in analytics-tools that this uses which can be deleted. https://github.com/edx/analytics-tools/tree/master/bigquery-monitoring

Reply:
https://github.com/edx/analytics-tools/pull/48